src/status_im/events.cljs

```diff
@@ -1,65 +1,64 @@
 (ns status-im.events
-  (:require [re-frame.core :as re-frame]
-            [status-im.chat.models :as chat]
-            [status-im.i18n.i18n :as i18n]
-            [status-im.mailserver.core :as mailserver]
-            [status-im.multiaccounts.core :as multiaccounts]
-            [status-im.ui.components.react :as react]
-            [status-im.utils.fx :as fx]
-            status-im.utils.logging.core
+  (:require clojure.set
```

Comment:
why shuffle namespaces ?

Reply:
I just sorted them. The more we have the harder it is to quickly check whether necessary ns is required and we already had one duplication there. https://github.com/status-im/status-react/blob/develop/src/status_im/events.cljs#L24